tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/DatabaseMonitoringTests.cs

```diff
@@ -0,0 +1,108 @@
+// <copyright file="DatabaseMonitoringTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Datadog.Trace.ClrProfiler.AutoInstrumentation.AdoNet;
+using Datadog.Trace.TestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.AdoNet
+{
+    [Trait("RequiresDockerDependency", "true")]
+    public class DatabaseMonitoringTests : TracingIntegrationTest
+    {
+        public DatabaseMonitoringTests(ITestOutputHelper output)
+            : base("MySql", output)
+        {
+            SetServiceVersion("1.0.0");
+            SetEnvironmentVariable("DD_ENV", "testing");
+            SetEnvironmentVariable("DD_DBM_PROPAGATION_MODE", "full");
+        }
+
+        public static IEnumerable<object[]> GetMySql8Data()
+        {
+            foreach (object[] item in PackageVersions.MySqlData)
+            {
+                if (!((string)item[0]).StartsWith("8") && !string.IsNullOrEmpty((string)item[0]))
+                {
+                    continue;
+                }
+
+                yield return item;
+            }
+        }
+
+        public override Result ValidateIntegrationSpan(MockSpan span) => span.IsMySql();
+
+        // Check that the span has been tagged after the comment was propagated
+        private void ValidateDbmTaggedSpans(IEnumerable<MockSpan> spans)
+        {
+            foreach (var span in spans)
+            {
+                Console.WriteLine(span.ToString());
+                Assert.True(span.Tags?.ContainsKey(Tags.DbmDataPropagated));
+            }
+        }
+
+        [Fact]
+        private void ConfirmDbmPropagateSpanData()
```

Comment:
I think this unit test would be better placed in the `Datadog.Trace.Tests` project since it doesn't interact with the test applications. You'll also need to make these test cases `public` instead of `private`